inbox.js

```diff
@@ -38,32 +64,36 @@ function refresh_inbox(wallet) {
 			inbox_pane.removeChild(inbox_pane.firstChild);
 		}
 
-		mail_ids.forEach(async function(id) {
-			var tr = document.createElement("tr");
-			var td_name = document.createElement("td");
-			var td_from = document.createElement("td");
-			var td_fee = document.createElement("td");
-			var td_qty = document.createElement("td");
-			var link = document.createElement("a");
-			tr.appendChild(td_from);
-			tr.appendChild(td_name);
-			tr.appendChild(td_fee);
-			tr.appendChild(td_qty);
-			td_name.appendChild(link);
-
-			var tx = await this.arweave.transactions.get(id);
-			var tx_status = await this.arweave.transactions.getStatus(id);
-			td_from.innerHTML = await arweave.wallets.ownerToAddress(tx.owner);
-			td_fee.innerHTML = arweave.ar.winstonToAr(tx.reward);
-			td_qty.innerHTML = arweave.ar.winstonToAr(tx.quantity);
-
-			link.innerHTML = id;
-			link.onclick = function() { show_mail(id) };
-
-			inbox_pane.appendChild(tr);
-		});
+	function displayMails(){
 
-	})();
-}
+	mail_UnixTime.sort((a,b)=>(Number(b.UnixTime) - Number(a.UnixTime)));
```

Comment:
This place needs an update after the tag has been renamed. For consistency I would call it `unixTime`, here and where you assign it to `tx-row`.

Reply:
yes added two many dashed